abis_mapping/templates/survey_occurrence_data_v2/mapping.py

```diff
@@ -1560,6 +1554,9 @@ def add_observation_verbatim_id(
         graph.add((uri, rdflib.SOSA.hasResult, verbatim_id))
         graph.add((uri, rdflib.SOSA.hasSimpleResult, rdflib.Literal(row["verbatimIdentification"])))
         graph.add((uri, rdflib.SOSA.observedProperty, CONCEPT_TAXON))
+
+        # Declare temporal entity allowing for correct type assignments
+        temporal_entity: rdflib.term.Node | None = None
```

Comment:
when you are declaring a variable mainly to give it a certain type, 
and the variable is given a value in every following code path,
you can leave out the initial value, e.g.
```python
temporal_entity: rdflib.term.Node | None  # leave out = None
```
no need to change this, just a pro tip

Reply:
yeah i know, for whatever reason I decided to go this way.